source/sqlsrv/stmt.cpp

```diff
@@ -129,6 +129,10 @@ ss_sqlsrv_stmt::ss_sqlsrv_stmt( _In_ sqlsrv_conn* c, _In_ SQLHANDLE handle, _In_
     fetch_fields_count ( 0 )
 {
     core_sqlsrv_set_buffered_query_limit( this, SQLSRV_G( buffered_query_limit ) TSRMLS_CC );
+
+    // initialize date_as_string based on the corresponding connection option
+    ss_sqlsrv_conn* ss_conn = static_cast<ss_sqlsrv_conn*>(conn);
```

Comment:
What's `conn` here? The sqlsrv_conn getting passed into the constructor is named `c`.

Reply:
`conn` (ptr to `sqlsrv_conn`) is a member of `struct sqlsrv_stmt`